error/s2n_errno.c

```diff
@@ -415,3 +416,60 @@ int s2n_error_get_type(int error)
 {
     return (error >> S2N_ERR_NUM_VALUE_BITS);
 }
+
+
+/* https://www.gnu.org/software/libc/manual/html_node/Backtraces.html */
+static bool s_s2n_stack_traces_enabled;
+
+bool s2n_stack_traces_enabled()
+{
+    return s_s2n_stack_traces_enabled;
+}
+
+int s2n_stack_traces_enabled_set(bool newval)
+{
+    s_s2n_stack_traces_enabled = newval;
+    return S2N_SUCCESS;
+}
+
+#define MAX_BACKTRACE_DEPTH 20
+__thread char ** s2n_stacktrace = NULL;
+__thread int s2n_stacktrace_size = 0;
+
+int s2n_free_stacktrace(void)
+{
+    if(s2n_stacktrace != NULL) {
```

Comment:
nitpicks: we usually have a space after `if`, `for` and before `(`

Reply:
Done